js/src/components/customer-effort-score-prompt/index.js

```diff
@@ -37,6 +37,9 @@ import useEffectRemoveNotice from '.~/hooks/useEffectRemoveNotice';
  * A CustomerEffortScore wrapper that uses tracks to track the selected
  * customer effort score.
  *
+ * TODO: Remove Temoorary fix for WC 7.3 in which adding secondQuestion is required.
```

Comment:
We could also use our [compat code comment](https://automatap2.wordpress.com/2022/11/30/legacy-compatibility-code-comment/) 
```suggestion
 * compatibility-code "WC >= 7.3"
 * TODO: Remove temporary fix for WC 7.3 in which adding `secondQuestion` is required.
```